### ISSUE TITLE
can this merge different commits?

### ISSUE THREAD

**Issue by AiJWang** [ryansgot/android-java-coverage-merger#13]

**Comment by ryansgot** [comment]
No. This is a very old gradle plugin for merging test coverage reports for instrumentation and jvm test coverage for Android. It's likely outdated and irrelevant at this point.
